dev/examples/data.js

```diff
@@ -1707,6 +1708,47 @@ const DATA = [
             showVGrid: true,
             showHGrid: true
         })
+    },
+    {
+        DASH_LINE: (id) => ({
```

Comment:
Can we re-use existing object from `LINE_DEFAULT`? 
Add property directly in the example. This file will be removed eventually.

Reply:
Updated